visdag/watcher.json

```diff
@@ -0,0 +1,262 @@
+[
```

Comment:
Why is this file in the repo?

Reply:
Junk file left over from a git add on the directory. Removed.